[2024-01-28T23:59:12.997+0100] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: match_id_dag.fetch_match_ids manual__2024-01-28T22:55:31.117284+00:00 [queued]>
[2024-01-28T23:59:13.004+0100] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: match_id_dag.fetch_match_ids manual__2024-01-28T22:55:31.117284+00:00 [queued]>
[2024-01-28T23:59:13.004+0100] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-28T23:59:13.069+0100] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): fetch_match_ids> on 2024-01-28 22:55:31.117284+00:00
[2024-01-28T23:59:13.075+0100] {standard_task_runner.py:60} INFO - Started process 11432 to run task
[2024-01-28T23:59:13.078+0100] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'match_id_dag', 'fetch_match_ids', 'manual__2024-01-28T22:55:31.117284+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/match_id_dag.py', '--cfg-path', '/tmp/tmphag9u9lh']
[2024-01-28T23:59:13.080+0100] {standard_task_runner.py:88} INFO - Job 19: Subtask fetch_match_ids
[2024-01-28T23:59:13.597+0100] {task_command.py:423} INFO - Running <TaskInstance: match_id_dag.fetch_match_ids manual__2024-01-28T22:55:31.117284+00:00 [running]> on host ubuntu.ubuntu.virtualbox.org
[2024-01-28T23:59:13.771+0100] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='irachide1@gmail.com' AIRFLOW_CTX_DAG_OWNER='hadoop' AIRFLOW_CTX_DAG_ID='match_id_dag' AIRFLOW_CTX_TASK_ID='fetch_match_ids' AIRFLOW_CTX_EXECUTION_DATE='2024-01-28T22:55:31.117284+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-28T22:55:31.117284+00:00'
[2024-01-28T23:59:13.773+0100] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-01-28T23:59:13.774+0100] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/hadoop/Desktop/projectBD/scripts/fetch_match_ids.py ']
[2024-01-28T23:59:13.788+0100] {subprocess.py:86} INFO - Output:
[2024-01-28T23:59:28.267+0100] {subprocess.py:93} INFO - Successfully uploaded ..//Tarelcat_match_ids.json to /user/hadoop/lol/raw/match_ids
[2024-01-28T23:59:28.268+0100] {subprocess.py:93} INFO - Match IDs for Tarelcat saved locally and uploaded to HDFS.
[2024-01-28T23:59:28.268+0100] {subprocess.py:93} INFO - Successfully uploaded ..//Kawozz_match_ids.json to /user/hadoop/lol/raw/match_ids
[2024-01-28T23:59:28.268+0100] {subprocess.py:93} INFO - Match IDs for Kawozz saved locally and uploaded to HDFS.
[2024-01-28T23:59:28.268+0100] {subprocess.py:93} INFO - Successfully uploaded ..//Saken_match_ids.json to /user/hadoop/lol/raw/match_ids
[2024-01-28T23:59:28.268+0100] {subprocess.py:93} INFO - Match IDs for Saken saved locally and uploaded to HDFS.
[2024-01-28T23:59:28.290+0100] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-28T23:59:28.322+0100] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=match_id_dag, task_id=fetch_match_ids, execution_date=20240128T225531, start_date=20240128T225912, end_date=20240128T225928
[2024-01-28T23:59:28.411+0100] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-28T23:59:28.423+0100] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
